tests/resources/Harbor-Pages/Configuration.robot

```diff
@@ -235,63 +235,75 @@ Disable Read Only
-    Click Element  xpath=//button[contains(.,'New Label')]
+    Retry Element Click  xpath=//button[contains(.,'New Label')]
     Sleep  1
     Input Text  xpath=//*[@id='name']  ${labelname}
     Sleep  1
-    Click Element  xpath=//hbr-create-edit-label//clr-dropdown/clr-icon
+    Retry Element Click  xpath=//hbr-create-edit-label//clr-dropdown/clr-icon
     Sleep  1
-    Click Element  xpath=//hbr-create-edit-label//clr-dropdown-menu/label[1]
+    Retry Element Click  xpath=//hbr-create-edit-label//clr-dropdown-menu/label[1]
     Sleep  1
     Input Text  xpath=//*[@id='description']  global
-    Click Element  xpath=//div/form/section/label[4]/button[2]
+    Retry Element Click  xpath=//div/form/section/label[4]/button[2]
     Capture Page Screenshot
     Wait Until Page Contains  ${labelname}
 
 Update A Label
     [Arguments]  ${labelname}
-    Click Element  xpath=//clr-dg-row[contains(.,'${labelname}')]//clr-checkbox-wrapper
+    Retry Element Click  xpath=//clr-dg-row[contains(.,'${labelname}')]//clr-checkbox-wrapper
     Sleep  1
-    Click Element  xpath=//button[contains(.,'Edit')]
+    Retry Element Click  xpath=//button[contains(.,'Edit')]
     Sleep  1
     Input Text  xpath=//*[@id='name']  ${labelname}1
     Sleep  1
-    Click Element  xpath=//hbr-create-edit-label//form/section//button[2]
+    Retry Element Click  xpath=//hbr-create-edit-label//form/section//button[2]
     Capture Page Screenshot
     Wait Until Page Contains  ${labelname}1
 
 Delete A Label
     [Arguments]  ${labelname}
-    Click Element  xpath=//clr-dg-row[contains(.,'${labelname}')]//clr-checkbox-wrapper
+    Retry Element Click  xpath=//clr-dg-row[contains(.,'${labelname}')]//clr-checkbox-wrapper
     Sleep  1
-    Click ELement  xpath=//button[contains(.,'Delete')]
+    Retry Element Click  xpath=//button[contains(.,'Delete')]
     Sleep  3
     Capture Page Screenshot
-    Click Element  xpath=//clr-modal//div//button[contains(.,'DELETE')]
+    Retry Element Click  xpath=//clr-modal//div//button[contains(.,'DELETE')]
     Wait Until Page Contains Element  //clr-tab-content//div[contains(.,'${labelname}')]/../div/clr-icon[@shape='success-standard']
 
-## Garbage Collection	
+## Garbage Collection
 Switch To Garbage Collection
     Sleep  1
     Retry Element Click  xpath=${gc_config_page}
-    Wait Until Page Contains Element  ${garbage_collection_xpath} 
-    Click Element  xpath=${garbage_collection_xpath}
+    Wait Until Page Contains Element  ${garbage_collection_xpath}
+    Retry Element Click  xpath=${garbage_collection_xpath}
 
 Click GC Now
     Sleep  1
-    Click Element  xpath=${gc_now_xpath}
+    Retry Element Click  xpath=${gc_now_xpath}
     Sleep  2
 
 View GC Details
-    Click Element  xpath=${gc_log_details_xpath}
+    Retry Element Click  xpath=${gc_log_details_xpath}
     Sleep  2
 
 Switch To GC History
     Retry Element Click  xpath=${gc_log_xpath}
     Retry Wait Until Page Contains  Job
 
+Create A New System CVE Whitelist
```

Comment:
This is "Add Items To System CVE Whitelist"

Reply:
Yes, I will modify it.